modules/options/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -247,7 +247,12 @@ final case class BuildOptions(
 
   // used when downloading fails
   private def defaultStableScalaVersions =
-    Seq(defaultScala212Version, defaultScala213Version, defaultScalaVersion)
+    Seq(
+      defaultScala212Version,
+      defaultScala213Version,
+      defaultScalaVersion,
+      scalaOptions.defaultScalaVersion.getOrElse(defaultScalaVersion)
+    ).distinct
```

Comment:
This whole variable seems to be unused, I'd delete it

Reply:
Oh, you seem to be correct.
Thanks, deleted